src/main/clojure/clara/tools/testing_utils.clj

```diff
@@ -75,3 +75,9 @@
   (f)
   (with-redefs [uc/get-ordered-update-cache ca/get-cancelling-update-cache]
     (f)))
+
```

Comment:
This is a clj file, if you want to use a function in it in ClojureScript it will need to be turned into a cljc file.  Note the [failed tests in your Travis build](https://travis-ci.org/cerner/clara-rules/builds/264850588?utm_source=github_status&utm_medium=notification).